packages/ps/bin/run

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+const { Command, Errors } = require('@oclif/core')
```

Comment:
The more I look at this the more nervous I get. I don't know when this file gets called and what language features we can rely on.

Reply:
Once we get all of the packages updated, comprehensive testing will hopefully reveal any issues. We can also do manual testing of the CLI.